scripts/add_electricity.py

```diff
@@ -352,6 +360,9 @@ def update_transmission_costs(n, costs, length_factor=1.0):
     n.lines["capital_cost"] = (
         n.lines["length"] * length_factor * costs.at["HVAC overhead", "capital_cost"]
     )
+    n.lines["investment"] = (
```

Comment:
I am contemplating whether in PyPSA network the column should be called `overnight_cost`. This term might be less ambiguous than `investment`. For cleaning the `technology-data` CSV files, it is clear that it should be named `investment`.